### DISCUSSION TITLE
Official ARM compiler

### DISCUSSION THREAD

**Discussion by jonnygrant** [compiler-explorer/compiler-explorer#4961] [Ideas]
Has anyone tried to get the official ARM compiler on godbolt? Version 6 is the latest.

**Comment by AbrilRBS** [comment]
Is this https://developer.arm.com/Tools%20and%20Software/Arm%20Compiler%20for%20Embedded the one you mean?

**Reply by jonnygrant** [reply]
Hello @RubenRBS 
I believe that is the more modern one they based on Clang.  There is an older one called "armcc"

There's probably not much use in having _Yet Another Clang_ variant 👍

I believe this is armcc:

[https://developer.arm.com/documentation/dui0472/latest/](https://developer.arm.com/documentation/dui0472/latest/)

**Comment by partouf** [comment]
We did add a couple of the official GCC derived compilers until last year, see https://github.com/compiler-explorer/compiler-explorer/issues/2359
This is the first time I'm noticing the embedded compiler. It seems more complicated license-wise.

**Reply by jonnygrant** [reply]
@partouf  Many thanks for the reply and link to the issue.  Probably the official GCC one may be enough. I shared a link above to what they call "armcc". Perhaps they are phasing it out.  So many are basing on clang these days (ICC etc)
